lib/terrafying/components/vpn_oidc.rb

```diff
@@ -74,11 +74,19 @@ def create_in
           openvpn_service,
           openvpn_authz_service(@ca, @fqdn, @route_all_traffic, @route_dns_entries, @groups, @client_id, @issuer_url),
         ]
+        
+        
         files = [
           openvpn_conf,
           openvpn_env,
           openvpn_ip_delay,
         ]
+
+        if @ca
+          units + [cert_checking_service, cert_checking_path, cert_checking_timer,restart_openvpn_authz_service]
```

Comment:
This will not actually change the `units` as the `+` operation in ruby returns a copy of the array. `units += [...]` should do the trick.